test/core/bad_client/tests/large_metadata.cc

```diff
@@ -38,7 +38,7 @@
 #define PFX_TOO_MUCH_METADATA_FROM_CLIENT_REQUEST         \
   "\x00\x00\x00\x04\x01\x00\x00\x00\x00"                  \
   "\x00"                                                  \
-  "5{\x01\x05\x00\x00\x00\x01"                            \
+  "\x1aM\x01\x05\x00\x00\x00\x01"                         \
```

Comment:
can you explain this change please?

Reply:
The frame length was incorrect, meaning that this test would simply buffer and wait forever due to an incomplete request.